cypress/e2e/pages/vaults/[vaultid]/index.spec.ts

```diff
@@ -66,10 +64,10 @@ context("/vaults/[vaultid] on desktop", () => {
     });
 
     it("should have 4 cells in each row", () => {
-      cy.findAllByTestId("OverflowTable.Row").within(() => {
-        cy.findAllByTestId("OverflowTable.Cell")
-          .should("have.length", 4)
```

Comment:
I think we should re-add this validation to validate length/count 4. Unless there's any reason?

Reply:
reverted back, will keep an eye if it will be cause of the failing.